src/historyView/branchChangesProvider.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  TreeDataProvider,
+  Disposable,
+  TreeItem,
+  Event,
+  commands
+} from "vscode";
+import { Model } from "../model";
+import { ISvnPathChange, Status } from "../common/types";
+import { openDiff, getIconObject } from "./common";
+import { dispose } from "../util";
+
+export class BranchChangesProvider
+  implements TreeDataProvider<ISvnPathChange>, Disposable {
+  private _dispose: Disposable[] = [];
+
+  dispose() {
+    dispose(this._dispose);
+  }
+
+  onDidChangeTreeData?: Event<ISvnPathChange> | undefined;
```

Comment:
Is this used by anything?

Reply:
0a71c09f5eca2e712f3c2fa766ededa5e7d38bc9